website/docs/r/dnat.html.markdown

```diff
@@ -52,9 +52,9 @@ The following arguments are supported:
 * `port` - (Required) The port number to map. -1 translates to "any"
 * `translated_port` - (Optional) The port number to map
 * `internal_ip` - (Required) The IP of the VM to map to
-* `protocol` - (Optional; *v2.0+*) The protocol type. Possible values are TCP, UDP, TCPUDP, ICMP, ANY. TCP is default to be backward compatible with previous version
+* `protocol` - (Optional; *v2.0+*) The protocol type. Possible values are tcp, udp, tcpupd, icmp, any. `tcp` is default to be backward compatible with previous version
```

Comment:
Please quote all these values. Now that they are lower case they need emphasis.

Reply:
done